### ISSUE TITLE
make fails - ioctl_cfg80211.c:458:17: error: too many arguments

### ISSUE THREAD

**Issue by KC3DRU** [lwfinger/rtl8723du#32]
Ubuntu 22.04

rtl8723du/os_dep/linux/ioctl_cfg80211.c:458:17: error: too many arguments to function ‘cfg80211_ch_switch_started_notify’

**Comment by ghost** [comment]
I'm also having this issue on v5.13.4 branch. But when I use the master branch, it works fine. Maybe you should try clone the master branch and `make` again.

This is the full error log on v5.13.4 branch:
```bash
/home/hyu/Downloads/rtl8723du-5.13.4/os_dep/linux/ioctl_cfg80211.c: In function ‘rtw_cfg80211_ch_switch_notify’:
/home/hyu/Downloads/rtl8723du-5.13.4/os_dep/linux/ioctl_cfg80211.c:458:3: error: too many arguments to function ‘cfg80211_ch_switch_started_notify’
  458 |   cfg80211_ch_switch_started_notify(adapter->pnetdev, &chdef, 0, 0, false);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/hyu/Downloads/rtl8723du-5.13.4/include/osdep_service_linux.h:98,
                 from /home/hyu/Downloads/rtl8723du-5.13.4/include/osdep_service.h:54,
                 from /home/hyu/Downloads/rtl8723du-5.13.4/include/drv_types.h:27,
                 from /home/hyu/Downloads/rtl8723du-5.13.4/os_dep/linux/ioctl_cfg80211.c:17:
/usr/src/linux-headers-5.10.0-18-common/include/net/cfg80211.h:7525:6: note: declared here
 7525 | void cfg80211_ch_switch_started_notify(struct net_device *dev,
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[3]: *** [/usr/src/linux-headers-5.10.0-18-common/scripts/Makefile.build:291: /home/hyu/Downloads/rtl8723du-5.13.4/os_dep/linux/ioctl_cfg80211.o] Error 1
make[2]: *** [/usr/src/linux-headers-5.10.0-18-common/Makefile:1856: /home/hyu/Downloads/rtl8723du-5.13.4] Error 2
make[1]: *** [/usr/src/linux-headers-5.10.0-18-common/Makefile:185: __sub-make] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-5.10.0-18-amd64'
make: *** [Makefile:2462: modules] Error 2
```

**Comment by lwfinger** [comment]
That was my error. I started the 4-argument version with 5.10. It should have been 5.11.
